rusoto/credential/src/profile.rs

```diff
@@ -43,6 +43,9 @@ pub struct ProfileProvider {
     file_path: PathBuf,
     /// The Profile Path to parse out of the Credentials File.
     profile: String,
+    /// If `true` disable [`credential_process`][credential_process] option, making sure not to
```

Comment:
Nit: should this be a link to the credential process docs?

Reply:
Actually, I don't know I just copied it from line 33. To be hoest, I have not yet figured out how to make working hyperlinks in Rust doc comments.

Reply:
Perhaps the best option for this is to link to https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#sourcing-credentials-from-external-processes like https://docs.rs/rusoto_credential/0.42.0/rusoto_credential/struct.ProfileProvider.html does. 

https://github.com/rusoto/rusoto/blob/master/rusoto/credential/src/profile.rs#L29-L39 shows how that link is done. 👍